hr_holidays_working_time/models/hr_holidays.py

```diff
@@ -55,7 +55,8 @@ def _auto_init(self, cr, context=None):
     number_of_hours = fields.Float(
         'Number of Hours', compute='_compute_number_of_hours', store=True)
     number_of_days_temp = fields.Float(
-        compute='_compute_number_of_hours_from_hours', store=True)
+        compute='_compute_number_of_hours_from_hours', store=True,
+        required=False)
```

Comment:
after checking some Odoo code it appears that actually required attribute is  quite useless into computed field but as it is a mutation from a Float to a computed the required attribute is conserved from the native field declaration. So I don't think it is an Odoo bug. Concerning the constraint is it really necessary as the computing result can not be driven into a False computed value by the added algorithm?